exawind/Pazy_Wing/README.md

```diff
@@ -0,0 +1,177 @@
+
+**Contents**
+
+- [Simulation Description and Setup](#simulation-description-and-setup)
+- [Postprocessing](#postprocessing)
+- [Code Performance](#code-performance)
+- [Results](#results)
+
+## Simulation Description and Setup
+
+The geometry and flow conditions of the Pazy Wing case are defined in Avin et al. [1]  They are summarized in the following table.
+
+| Case Property     | Value                  |
+| :---              | ---                    |
+| Span              | 0.55 m                 |
+| Chord             | 0.10 m                 |
+| Airfoil           | NACA0018               |
+| Angle of Attack   | 3-7 deg                |
+| Density           | 1.225 kg/m<sup>3</sup> |
+| Dynamic Viscosity | 1.8E-05 Pa-s           |
+| Velocity          | 15-50 m/s              |
+| Mach Number       | 0.04-0.15              |
+| Reynolds Number   | < 3.4E+05              |
+| Turbulence Level  | 0.5%                   |
+
+Note that the wind-tunnel model given in [1] has several small unintentional features, including varying airfoil thickness along the span, sag in the flap-wise direction, and twist (about 1 degree).  It is assumed here that these features have a negligible impact on the FSI results, and an idealized version of the wing is used instead, with a uniform cross-section along the span.
+
+**Domain Overview**
+
+<img src="figures/Pazy_Overview_Fig_550.png" border="0" alt=""/>
+
+The CFD computational domain consists of an inner Nalu-Wind domain immediately surrounding the wing, and an outer AMR-Wind domain to allow space for the wake to propagate.  The sides, ceiling, and floor of the AMR-Wind domain are intended to approximate the effect of the wind tunnel walls used in the experiment.  However, to avoid the computational expense of resolving the boundary layers near the tunnel walls, *slip wall* boundaries are used there.  The Nalu-Wind domain extends a few chord-lengths away from the wing, and terminates with an *overset* boundary on the exterior, and a *slip wall* on the portion that touches the bottom wind tunnel boundary.  The inner boundary of the Nalu-Wind domain represents the wing surface, and uses a *no-slip wall* boundary condition.
+
+In the experiment, the blade tip passes near the wind-tunnel walls at peak deflection.  Since the Nalu-Wind domain extends several chord lengths away from the wing, the AMR-Wind domain was chosen to be somewhat larger than the wind tunnel in order to avoid collisions of the overset boundary with the slip wall at peak deflection, and additional padding was provided to ensure the results are not contaminated by the effect of the artificial slip wall boundaries.  Future studies are planned to investigate the impact of the choice of AMR-Wind domain size on the simulation results.
+
```

Comment:
Maybe we can add a short section here about how the structural dynamics of the blade are modeled using OpenFAST, and the aero-elastic coupling is between Nalu-Wind and OpenFAST, with AMR-Wind interfacing just with Nalu-Wind as the background solver.  This is just to avoid any confusion with people who saw the NREL5MW ALM case and thought aeroelastic coupling was between AMR-Wind/OpenFAST.